src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.figures;
+
+public class Circle extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line before the class declaration.